### PR TITLE
Edit: Getting coins

### PR DESCRIPTION
**Pull request by Mocab**

### CODE REVIEW

File: _posts/falix/general/2021-08-10-getting-coins.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ You can refer your friend to register through your affiliate for 50 coins per us
 ## Joining Promoted Discord Servers
 
 You can join a promoted server and receive FalixCoins for it! Simply go to the [promotions page](https://client.falixnodes.net/join_for_rewards) to see the promoted servers available along with the expiry date and the number of coins you will receive from joining it.
+
+Do note that you must log out and login to recieve your coins.
```
Comment (itsjuoum) [inline _posts/falix/general/2021-08-10-getting-coins.md:21]:
```suggestion
Note that you must log out and log back in to receive your coins.
```

**Review by itsjuoum — CHANGES_REQUESTED** [review]

**Comment by DaddyChimken** [comment]
Wasn't a "Refresh Guilds" (or something similar) button added? Thus making logging out unnecessary.

**Comment by Mocab** [comment]
> Wasn't a "Refresh Guilds" (or something similar) button added? Thus making logging out unnecessary.

I do not remember seeing it anywhere, I will double check after a while

File: _posts/falix/general/2021-08-10-getting-coins.md [hunk 2]
```diff
@@ -17,3 +17,7 @@ You can refer your friend to register through your affiliate for 50 coins per us
 ## Joining Promoted Discord Servers
 
 You can join a promoted server and receive FalixCoins for it! Simply go to the [promotions page](https://client.falixnodes.net/join_for_rewards) to see the promoted servers available along with the expiry date and the number of coins you will receive from joining it.
+
+To receive your coins for joining servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the join for resources page.
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-08-10-getting-coins.md:21]:
```suggestion
To receive your FalixCoins from joining promoted servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the join for resources page.
```

Please also add a hyperlink to the join for resources page.

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/falix/general/2021-08-10-getting-coins.md [hunk 3]
```diff
@@ -17,3 +17,7 @@ You can refer your friend to register through your affiliate for 50 coins per us
 ## Joining Promoted Discord Servers
 
 You can join a promoted server and receive FalixCoins for it! Simply go to the [promotions page](https://client.falixnodes.net/join_for_rewards) to see the promoted servers available along with the expiry date and the number of coins you will receive from joining it.
+
+To receive your FalixCoins from joining promoted servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the [join for resources page](https://client.falixnodes.net/join_for_rewards).
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-08-10-getting-coins.md:21]:
```suggestion
To receive your FalixCoins from joining promoted servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the [join for rewards page](https://client.falixnodes.net/join_for_rewards).
```

**Review by DaddyChimken — APPROVED** [review]

**Review by itsjuoum — APPROVED** [review]

**Pull request merged**